conngen/conngenmodule.cpp

```diff
@@ -106,48 +104,29 @@ ConnectionGeneratorModule::init( SLIInterpreter* i )
    If not specified, the synapse model is taken from the Options of
    the Connect command.
 
+   Availability: Only if compiled with libneurosim support
    Author: Jochen Martin Eppler
    FirstVersion: August 2012
    SeeAlso: Connect, synapsedict, GetOptions, CGParse, CGParseFile,
-   CGSelectImplementation, cgstart,
-   cgsetmask, cgnext
+   CGSelectImplementation
 */
 
-// Connect for conn_generator subnet subnet dict synapsetype
+// CGConnect for conngen gidcollection gidcollection dict literal
```

Comment:
It might be a convention, but adding the parameter types of a function causes tight coupling: If you want to change the interface you have to change the name. The Api will get really unstable this way

Reply:
You are right. However, this is indeed a convention that makes it much easier to get things right on the SLI level. The documentation for the type codes can be found [here](https://github.com/nest/nest-simulator/blob/master/nestkernel/nestmodule.h#L60).